amundsen_application/static/js/components/Tags/TagInfo/index.spec.tsx

```diff
@@ -5,11 +5,17 @@ import * as React from 'react';
 import { shallow } from 'enzyme';
 
 import * as UtilMethods from 'ducks/utilMethods';
+import { updateSearchState } from 'ducks/search/reducer';
+
 import { mapDispatchToProps, TagInfo, TagInfoProps } from '.';
 
 const logClickSpy = jest.spyOn(UtilMethods, 'logClick');
 logClickSpy.mockImplementation(() => null);
 
+jest.mock('ducks/search/reducer', () => ({
```

Comment:
Our testing framework `Jest` requires that any methods we will run an assertion on needs to be be mocked. This syntax is one way to mock imported methods, and you only have to mock the ones that will be used in the test file.